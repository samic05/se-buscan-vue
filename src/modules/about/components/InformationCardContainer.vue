<template>
  <div class="info">
    <h2 class="title">¿Por qué Se Buscan es importante?</h2>
  </div>
  <div class="card-container">
    <InformationCard
      v-for="card in cards"
      :key="card.title"
      :information="card"
    />
  </div>
</template>

<script>
import InformationCard from "./InformationCard.vue";

export default {
  components: {
    InformationCard,
  },
  data() {
    return {
      cards: [
        {
          image: require("@/assets/about-icons/social-network.svg"),
          title: "Red Social",
          content:
            "Para la comunidad de desaparecidos en el mundo. Sus familiares o amigos podrán vincularse todas las entidades involucradas en la problemática de desaparición. SE BUSCAN será el nodo mundial de información dedesaparecidos.",
        },
        {
          image: require("@/assets/about-icons/thanks.svg"),
          title: "Gratuita",
          content:
            "El acceso a la página WEB y a las aplicaciones en Android y Apple es totalmente gratuita y  a través de estos medios se podrán registrar las personas desaparecidas o enviar pistas.",
        },
        {
          image: require("@/assets/about-icons/worldwide.svg"),
          title: "Cubrimiento Mundial",
          content:
            "SE BUSCAN tiene cubrimiento mundial, es decir, que desde cualquier lugar del planeta con acceso a internet, se podrán registrar desaparecidos.",
        },
      ],
    };
  },
};
</script>

<style scoped>
.info {
  padding: 50px;
  margin-left: 100px;
}

.title {
  font-family: "Poppins", sans-serif;
  margin: 0 0 10px;
  color: #1d98ad;
}
.card-container {
  display: flex;
  justify-content: center;
  padding: 50px;
}
</style>
