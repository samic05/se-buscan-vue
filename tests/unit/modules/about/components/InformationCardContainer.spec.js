import InformationCardContainer from '@/modules/about/components/InformationCardContainer'
import { mount } from "@vue/test-utils"

describe('InformationCardContainerComponent',() => {

    test('should match with snapshot', () => {
        const informationCardContainerComponentWrapped =  mount(InformationCardContainer,{
            data () {
                return {
                    cards: [
                        {
                            image: require("@/assets/about-icons/social-network.svg"),
                            title: "Red Social",
                            content:
                              "Para la comunidad de desaparecidos en el mundo. Sus familiares o amigos podrán vincularse todas las entidades involucradas en la problemática de desaparición. SE BUSCAN será el nodo mundial de información dedesaparecidos.",
                          }
                    ]
                }
            }
        });
 
     expect(informationCardContainerComponentWrapped.html()).toMatchSnapshot()
 
     })
})