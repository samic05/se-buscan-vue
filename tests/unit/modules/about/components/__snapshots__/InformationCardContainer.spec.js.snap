// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InformationCardContainerComponent should match with snapshot 1`] = `
<div class="info">
  <h2 class="title">¿Por qué Se Buscan es importante?</h2>
</div>
<div class="card-container">
  <div class="card"><img src="" class="avatar" alt="Avatar">
    <div class="container">
      <h4><b>Red Social</b></h4>
      <p>Para la comunidad de desaparecidos en el mundo. Sus familiares o amigos podrán vincularse todas las entidades involucradas en la problemática de desaparición. SE BUSCAN será el nodo mundial de información dedesaparecidos.</p>
    </div>
  </div>
</div>
`;
